.eslintrc.json

```diff
@@ -1,19 +1,18 @@
 {
     "env": {
-        "browser": true,
-        "es6": true
-    },
-    "extends": [
-        "airbnb-base"
-    ],
-    "globals": {
-        "Atomics": "readonly",
-        "SharedArrayBuffer": "readonly"
+      "browser": true,
+      "es6": true,
+      "jest": true
     },
     "parserOptions": {
-        "ecmaVersion": 2018,
-        "sourceType": "module"
+      "ecmaVersion": 2018,
+      "sourceType": "module"
     },
+    "extends": ["airbnb-base"],
     "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "arrow-parens": [2, "as-needed", { "requireForBlockBody": true }]
```

Comment:
Hey @bolah2009, how are you? I had to put this rule because there is a mismatch when Stickler checks my project without it. On Github, I get an error because I'm using parentheses in an arrow function's single parameter but locally I don't get that error. It seems that Stickler uses a default configuration underground along with our configuration. Please, check the thread on Slack where we were talking about the plugin error I had.

Reply:
Yes @efrapp we spoke about this earlier. Could you open a PR on the linter config repo?

Reply:
@bolah2009 can we talk that on Slack? I checked a PR Jorge Rincón did about that so it would be great if all the stakeholders are aware of this.

Reply:
ok